Main.cpp

```diff
@@ -191,43 +239,63 @@ int main() {
       // canBus.printRxAll();
     }
 
-    chThdSleepMilliseconds(50);
+    chThdSleepMilliseconds(10000);
   }
 }
 
-// module utility functions
 // @brief Convert bytes in RX buffer from ADC reading to temperature
 // @return temp Temperature in degrees C, resolution of .25
+// TODO: Add in per-channel moving average
 uint8_t adc_to_temp(uint8_t* rxbuf) {
   static uint16_t raw{};
-  static int norm{};
+  static double voltage{};
+  static int lookup_index{};
```

Comment:
An unsigned will cause negative integers to wrap around prior to clamping. If there's a version of clamp that doesn't use references then I could use that